src/Kernel-BytecodeEncoders/EncoderForSistaV1.class.st

```diff
@@ -485,6 +485,15 @@ EncoderForSistaV1 class >> isCreateFullBlock: compiledBlock code: code at: pc [
 				compiledBlock = (code literalAt: (code at: pc + nExtBytes + 1) + (extA bitShift: 8) + 1)] ]
 ]
 
+{ #category : #'instruction stream support' }
+EncoderForSistaV1 class >> isCreateFullBlockAt: pc in: code [
+	^ self
+		  extensionsAt: pc
+		  in: code
+		  into: [ :extA :extB :nExtBytes | 
+		  (code at: pc + nExtBytes) = 16rF9 ]
```

Comment:
Why we can't put this logic into isCreateBlockAt:in: ? It would be good to add a comment about it

Reply:
I am not yet at all sure where to put what.. I think it is good to have, on the level of the encoder, two different methods to check (ad we have two different ways to create blocks).